invenio_previewer/templates/semantic-ui/invenio_previewer/base.html

```diff
@@ -0,0 +1,35 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2019 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+<!DOCTYPE html>
+<html {% block html_tags %}{% endblock %}>
+  <head>
+    <title>{{_('Preview')}}</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    {% block head %}
+      {% if config.PREVIEWER_ASSETS_USE_WEBPACK %}
```

Comment:
I deleted from here the webassets but kept the `PREVIEWER_ASSETS_USE_WEBPACK` for now. We will need to get rid of this as we support only webpack build now but I was waiting for confirmation. I will create a ticket related to that.